project.clj

```diff
@@ -40,7 +41,11 @@
   ;; code that we have.
   :classifiers [["test" :testutils]]
 
-  :profiles {:dev {:source-paths ["examples/shutdown_app/src"
+  :profiles {:cljfmt {:plugins [[lein-cljfmt "0.3.0"]
```

Comment:
There's newer versions of this plugin, but every project that I found in PL that has implemented this feature has used this version (its the version used in the example).